mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfiguration.java

```diff
@@ -206,7 +208,9 @@ public void setResourceLoader(ResourceLoader resourceLoader) {
 
     @PostConstruct
     public void afterPropertiesSet() {
-      log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      if (log.isDebugEnabled()) {
+        log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      }
```

Comment:
@kazuki43zoo I don't really see any savings in this specific one.  I understand the prior want to check for debug since there is an otherwise expensive looping on logging.  However, that isn't the case here.  I don't think this is going to amount to much in savings.  Am I missing something?